### DISCUSSION TITLE
Options drop down with new SQL Connector

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#1877] [Help] [answered]
Hi Is there a way to retain the functionality of the drop-down options menu in forms when using the new external DB SQL connector? basically, is there a way to have a option picker drop-down menu when writing directly to an external SQL DB?

**Comment by mike12345567** [comment, marked as answer]
Hi @AndrewJRamay this isn't something that can be done right now, but its a good idea, I will raise it with the team as I can see it being a very useful feature the ability to restrict an input to an SQL column.

**Reply by KernOil** [reply]
yes, that was a keystone feature when i was using the internal DB. Thank you

**Reply by mike12345567** [reply]
I've opened an issue to track this: https://github.com/Budibase/budibase/issues/1878

**Reply by KernOil** [reply]
Thank you very much!
